examples/getting-started-anylogic-sim/job.yml

```diff
@@ -0,0 +1,15 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+code: src
+command: >-
+  ./start.sh
```

Comment:
We have to change this to `bash start.sh`

When using `./start.sh`, no error context is given for debugging when things go wrong. The error logs only state "/bin/bash: ./start.sh: /usr/bin/sh^M: bad interpreter: No such file or directory".

Reply:
The commit `Add .gitattributes to force LF endings for some files` should solve this issue.

Note that I squashed some commit. You'll have to remove the old version of my branch you have locally and then pull again.
Before sending the job again to AML ensure ``start.sh`` has LF line ending instead of CRLF. You can check that at the bottom of VSCode when opening the file. If it's still CRLF checkout main and checkout back again my branch.

Reply:
Resolving this conversation as per our conversation earlier today. The two messages are equivalent, and it is related to an issue with the end of line character (CRLF vs LF). Thanks for clarifying it.